src/wanderer.ts

```diff
@@ -11,12 +23,13 @@ import {
   Guzzlr,
   have,
   SourceTerminal,
+  sumNumbers,
 } from "libram";
 import { estimatedTurns } from "./embezzler";
 import { globalOptions, HIGHLIGHT, propertyManager, realmAvailable } from "./lib";
 import { garboValue } from "./session";
 
-export type DraggableFight = "backup" | "wanderer";
+export type DraggableFight = "backup" | "wanderer" | "yr";
```

Comment:
Might want to make this an enum at this point; I think we should also go ahead and call this `"yellow ray"` as well.